root/static/scripts/common.js

```diff
@@ -1,8 +1,10 @@
-window.aclass = require("aclass");
-window.ko = require("knockout");
-window.L = require("leaflet");
-window._ = require("lodash");
-window.$ = window.jQuery = require("jquery");
+import global from './global';
```

Comment:
The name of the global object is different depending on the environment (`global` vs. `window`). I added a helper module to decide which to use.